packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

```diff
@@ -264,7 +264,59 @@ See [request properties](#objects-in-the-request-from-okta) for full details.
 }
 ```
 
-## Sample Progressive Enrollment request
+## Sample SSR responses
+
+The external service responds to Okta indicating whether to accept the end user's self-registration attempt. The response returns a `commands` object in the body of the HTTPS response. This object contains specific syntax that indicates whether the user is allowed or denied to self-register.
+
+See [response properties](#response-objects-that-you-send) for full details.
+
+### Sample SSR response using data.userProfile
+
+The following sample response shows a successful self-registration using a valid email domain:
```

Comment:
```suggestion
The following sample response shows a successful self-registration:
```

Reply:
I'd remove the "valid email domain"; that was the example we used for the guide, but it could revolve around any logic on the external server.